File: sdkwork-app-sdk-typescript/src/types/wallet-withdrawal-form.ts
/** 钱包提现请求 */
export interface WalletWithdrawalForm {
  /** 提现金额 */
  amount: number;
  /** 提现方式: ALIPAY/WECHAT/BANK */
  withdrawMethod: string;
  /** 收款账户 */
  accountNo?: string;
  /** 收款账户名称 */
  accountName?: string;
  /** 开户行 */
  bankName?: string;
  /** 幂等请求号(建议前端传UUID) */
  requestNo?: string;
  /** 备注 */
  remarks?: string;
}

File: sdkwork-app-sdk-go/types/wallet_withdrawal_form.go
package types

// 钱包提现请求
type WalletWithdrawalForm struct {
	Amount float64 `json:"amount"`
	WithdrawMethod string `json:"withdrawMethod"`
	AccountNo string `json:"accountNo"`
	AccountName string `json:"accountName"`
	BankName string `json:"bankName"`
	RequestNo string `json:"requestNo"`
	Remarks string `json:"remarks"`
}
